.env.example

```
@@ -5,6 +5,8 @@
 BREVO_API_KEY=your_brevo_api_key_here
 BREVO_NEWSLETTER_LIST_ID=your_list_id_here
 BREVO_WEBHOOK_SECRET=your_webhook_secret_here
+BREVO_SENDER_EMAIL=noreply@yourdomain.com # Must be verified in Brevo
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The BREVO_SENDER_EMAIL key should go before the BREVO_WEBHOOK_SECRET key

(UnorderedKey)